lnbits/extensions/discordbot/config.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Discord Bot",
+    "short_description": "Generate users and wallets",
```

Comment:
Maybe add a few more details to the description.

Reply:
Got it, thx.